src/Datagrid/Pager.php

```diff
@@ -40,6 +40,10 @@ abstract class Pager implements \Iterator, \Countable, \Serializable, PagerInter
     protected $lastPage = 1;
 
     /**
+     * NEXT_MAJOR: Remove this property and add a private "countResults" property.
```

Comment:
```suggestion
     * NEXT_MAJOR: Remove this property
```

If we don't implement countResult.